docker-compose.circle.yml

```diff
@@ -94,31 +94,31 @@ centos7build:
 ## Package testing nodes
 #
 wheezytest:
-  image: quay.io/dennybaa/droneunit:wheezy-sshd
+  image: stackstorm/packagingtest:wheezy
   extends:
     file: docker-compose.override.yml
     service: volumes-circle
 
 jessietest:
-  image: quay.io/dennybaa/droneunit:jessie-sshd
+  image: stackstorm/packagingtest:jessie
   extends:
     file: docker-compose.override.yml
     service: volumes-circle
 
 trustytest:
-  image: quay.io/dennybaa/droneunit:trusty-upstart
+  image: stackstorm/packagingtest:trusty
   extends:
     file: docker-compose.override.yml
     service: volumes-circle
 
 centos6test:
-  image: quay.io/dennybaa/droneunit:centos6-sshd
+  image: stackstorm/packagingtest:centos6
   extends:
     file: docker-compose.override.yml
     service: volumes-circle
 
 centos7test:
-  image: quay.io/dennybaa/droneunit:centos7-systemd
+  image: stackstorm/packagingtest:centos7
```

Comment:
> For me sshd is SSH server. How it relates to init systems, I have no idea.

You're right, it runs SSH server as main process, that's why it's called `jessie-sshd`, see [Dockerfile](https://github.com/dennybaa/dockerfiles/blob/master/droneunit/jessie/sshd/Dockerfile#L8): 
```sh
CMD [ "/usr/sbin/sshd", "-D" ]
```
By this rule `centos7-systemd` container runs [systemd as a main process](https://github.com/dennybaa/dockerfiles/blob/master/droneunit/centos7/systemd/Dockerfile#L25) and `trusty-upstart` runs upstart system as main PID: 1 process.

This naming is here, because it's _very unnatural_ to run init system inside the container, allowing to start/stop many services and use container like a real VM (you know that Docker is usually about 1 process per container).

If we name container just `centos7`, - it will look like the official one [centos7](https://hub.docker.com/_/centos/) and *nothing* will indicate that this image has any init system installed.

Will you figure that out if we omit that naming? Will it be harder/easier to understand if we omit that naming?
How about 6 months from now? I won't remember anything for sure.
There are just best practices in naming and ideally, I would like if we follow them for less confusion and time wasting in future.